easypermissions/build.gradle

```diff
@@ -33,14 +33,16 @@ junitJacoco {
 }
 
 dependencies {
-    api "com.android.support:appcompat-v7:$support_library_version"
-    api "com.android.support:support-compat:$support_library_version"
-    api "com.android.support:support-fragment:$support_library_version"
+    api "androidx.appcompat:appcompat:1.0.2"
+    api "androidx.core:core:1.0.1"
+    api "androidx.fragment:fragment:1.0.0"
 
     testImplementation 'junit:junit:4.12'
     testImplementation 'com.google.truth:truth:0.42'
     testImplementation 'org.robolectric:robolectric:4.1'
-    testImplementation 'org.robolectric:shadows-supportv4:4.1'
+    // TODO
+//    testImplementation 'org.robolectric:shadows-supportv4:4.1'
+    testImplementation 'androidx.fragment:fragment-testing:1.1.0-alpha03'
```

Comment:
Maybe use the previous stable version instead of the alpha one?

Reply:
I actually don't see any stable versions ever released on maven.google.com ... seems like `1.1.0-alpha01` was the first.  However I don't mind taking an `alpha` dependency in test only, although I agree with you that stable would be better.